### DISCUSSION TITLE
[Bug]

### DISCUSSION THREAD

**Discussion by DocLockusa** [webstudio-is/webstudio-community#202] [Q&A]
The webhook form is not working in mobile version

**Comment by kof** [comment]
Please provide more information. What steps did you do. What project that is? Please share the site url.

**Reply by kof** [reply]
Or even better - share a separate demo project where we can reproduce this and see its not working

**Comment by DocLockusa** [comment]
https://savingamericamoney.com

I used template tested it when it was still part of Webstudio but now as I
added my domain the contact webhook forms are not working. Do I need to
make a change to the settings or something?


Tj Berkley
(602) 390-0929

**Reply by kof** [reply]
I am confused now, is it not working at all or only on mobile as you initially mentioned?

**Comment by DocLockusa** [comment]
After I checked online again it says the emails are going thru but no notifications pushed thru. So I would say all are not working.
________________________________

**Reply by kof** [reply]
Just created a fresh webfhook form and it works. Pls try adding a new webhook form on the same site and see if that works

**Comment by DocLockusa** [comment]
Okay thank you

Tj Berkley
(602) 390-0929

**Comment by DocLockusa** [comment]
Question. If I wanted the webhook form to redirect to another page or link
after they submit, how would I do that?

Tj Berkley
(602) 390-0929

**Reply by kof** [reply]
https://github.com/webstudio-is/webstudio/issues/3530#issuecomment-2317596011 workaround at the moment

**Comment by kof** [comment]
closing this as doesn't seem to be a bug
